feature/settings/src/main/java/dev/anilbeesetti/nextplayer/settings/screens/player/PlayerPreferencesScreen.kt

```diff
@@ -483,6 +487,22 @@ fun AutoplaySetting(
     )
 }
 
+@Composable
+fun PipSetting(
+    isChecked: Boolean,
+    onClick: () -> Unit
+) {
+    PreferenceSwitch(
+        title = stringResource(id = R.string.pip_settings),
```

Comment:
R.string.pip_settings doesn't exist

Reply:
Yeah I will doing a fix! In a hurry didn't commit file 😅